<?xml version="1.0" ?>
<launch>

	<arg name="path_sensor_model"   value="$(find objects)/meshes/plug/config/X.txt"/>

	<node name="kuka_server_node" pkg="peg_hole_kuka" type="peg_kuka_server_node" required="true" output="screen">
		<param name="action_server_name" 		value="$(arg action_server_name)"/>
  		<param name="simulation" 			value="false" />
  		<param name="world_frame" 			value="/world" />
  		<param name="model_dt" 				value="0.01" />
  		<param name="reachingThreshold" 		value="0.01" />
  		<param name="orientationThreshold" 		value="0.02" />
		<param name="path_sensor_model" 		value="$(arg path_sensor_model)"/>
	</node>


</launch>
